cluster/manifests/flannel/configmap.yaml

```diff
@@ -1,30 +0,0 @@
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: kube-flannel-cfg
-  namespace: kube-system
-  labels:
-    application: kubernetes
-    component: flannel
-data:
-  cni-conf.json: |
-    {
-      "name": "podnet",
-      "cniVersion": "0.3.1",
-      "plugins": [
-        {
-          "type": "flannel",
-          "delegate": {
-            "isDefaultGateway": true,
-            "hairpinMode": true
-          }
-        }
-      ]
-    }
-  net-conf.json: |
-    {
-      "Network": "10.2.0.0/15",
-      "Backend": {
-        "Type": "vxlan"
-      }
-    }
```

Comment:
Added in the next commit. Thanks!

Reply:
I'm un-resolving this because we need to check if we can do that in this PR or have to do it in a subsequent PR. Especially with the recent karpenter changes that change the order of the rollout.

Reply:
Let's merge the PR, then we can check in `dev` channel if the `deletions.yaml` worked. If not I can create a new PR with the changes 🙂